src/makeInfinityAdder.js

```diff
@@ -1,10 +1,24 @@
 'use strict';
 
-/***
+/*
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(number) {
+    if (number === undefined) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    } else {
+      sum += number;
```

Comment:
Ensure that `number` is a valid number before adding it to `sum`. This will prevent potential issues if non-numeric values are passed to the `adder` function.